### DISCUSSION TITLE
Manca Icona Mail in elenco fatture

### DISCUSSION THREAD

**Discussion by baldioli** [fattureincloud/api#187] [Q&A] [answered]
Sto utilizando la versione 2 delle api per inviare a fattureincloud delle fatture di clienti occasionali.
Anche se i clienti sono occasionali avrei bisogno che l'elenco delle fatture inviate facciano vedere l'iconcina della mail.
Ho vosto però che questo capita solo se nella chiave Entity della fattura valorizzo il campo id.
Questo mi costringe ad importare anche il cliente, E' possibile vedere l'icona della mail nell'elenco fatture anche senza importare il cliente, considerando che il campo Entity / Mail io lo mando compilato ?

**Comment by emajo** [comment, marked as answer]
Ciao @baldioli purtroppo non è possibile in quanto anche se viene passato il campo email in creazione di fattura questo viene ignorato, l'unica soluzione è appunto quella di creare il cliente settando la email e passare l'id in creazione fattura.

Grazie,
Emanuele.
